core/base/libs/socrate/socrate/system.py

```diff
@@ -66,7 +66,7 @@ def _is_compatible_with_hardened_malloc():
         lines = f.readlines()
         for line in lines:
             # See #2764, we need vmovdqu
-            if line.startswith('flags') and ' avx ' not in line:
+            if line.startswith('flags') and ' avx ' not in line and ' avx2 ' not in line:
```

Comment:
```suggestion
            if line.startswith('flags') and ' avx2 ' not in line:
```
You won't find a CPU that does the later but not the former.

As currently spelt out, if either is present the test will skip. This is not what we want, we want both to be present.

Reply:
We may want to explain in the warning line 83 that we are missing vectorized instructions